lib/execution-helper.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Recursively walks through a folder and its children,
+ * calling a callback on each file
+ *
+ * @param {String} dir - directory to walk through
+ * @param {Function} callback - operation to perform on each file
+ */
+function walkDir(dir, callback) {
+  fs.readdirSync(dir).forEach((f) => {
+    const dirPath = path.join(dir, f);
```

Comment:
This is not necessarily a directory, so I think `directoryEntryPath` is probably better